tests/test_clockin.py

```diff
@@ -87,3 +88,33 @@ def test_get_national_holidays(self, login, auth_user_id, absences):
 
         assert len(national_holidays) == 1, 'National holidays should only have one day'
         assert national_holidays[0] == date(2019, 8, 19)
+
+
+class TestClockIn(object):
+
+    def test_create_friday_register(self):
+        clock_in = ClockIn(2021, 2)
+
+        with mock.patch('clockin.ClockIn._reduced_day') as mock_create_register, \
+                mock.patch.object(Absence, 'is_friday', new_callable=mock.PropertyMock, return_value=True):
```

Comment:
Why do you are mock is_friday? you can use a day that you know is friday

Reply:
Changed